docs/user-guide/common-use-cases/pab.md

```diff
@@ -1,42 +1,31 @@
-
-In order to create new wallet in a first place we need to invoke [create wallet](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postWallet) endpoint, for instance:
-
-```
-$ curl -X POST http://localhost:8090/v2/wallets \
--d '{"mnemonic_sentence":["slab","praise","suffer","rabbit","during","dream","arch","harvest","culture","book","owner","loud","wool","salon","table","animal","vivid","arrow","dirt","divide","humble","tornado","solution","jungle"],
-     "passphrase":"Secure Passphrase",
-     "name":"My Test Wallet",
-     "address_pool_gap":20}' \
--H "Content-Type: application/json"
-```
+## Overview
+This guide is to show how to invoke Plutus contracts with cardano-wallet.
 
 ## Workflow
 Once you have created a smart contract with [PAB](https://github.com/input-output-hk/plutus/tree/master/plutus-pab) you can execute it via cardano-wallet.
 
 There are three endpoints that need to be invoked to follow the workflow:
 
- - [Balance](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/balanceTransaction) - for balancing transaction from PAB.
- - [Sign](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/signTransaction) - for signing transaction.
- - [Submit](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/postExternalTransaction) - for submitting transaction to the network.
+ - [POST /wallets/{walletId}/transactions-balance](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/balanceTransaction) - for balancing transaction from PAB.
+ - [POST /wallets/{walletId}/transactions-sign](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/signTransaction) - for signing transaction.
+ - [POST   
+/wallets/{walletId}/transactions-submit](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/submitTransaction) - for submitting transaction to the network.
 
 ### Balance transaction
 
-Plutus Application Backend provides payload of unbalanced transaction. This transaction needs to be balanced with wallet inputs such that it can be submitted to the network. The response from this endpoint returns `fee` and `coin_selection` of balanced transaction as well as CBOR-encoded `transaction` represented in base64 encoding. We will need that returned `transaction` value to pass on to [sign](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/signTransaction) endpoint.
+Plutus Application Backend provides payload of unbalanced transaction. This transaction needs to be balanced with wallet inputs such that it can be submitted to the network. The response from this endpoint returns `fee` and `coin_selection` of balanced transaction as well as CBOR-encoded `transaction` represented in base64 encoding. We will need that returned `transaction` value to pass on to [POST /wallets/{walletId}/transactions-sign](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/signTransaction) endpoint.
```

Comment:
```suggestion
Plutus Application Backend provides a payload of an unbalanced transaction. This transaction needs to be balanced with wallet's inputs such that it can be submitted to the network. The response from this endpoint returns `fee` and `coin_selection` of the balanced transaction as well as CBOR-encoded `transaction` represented in base64 encoding. We will need the returned `transaction` value to pass on to [POST /wallets/{walletId}/transactions-sign](https://input-output-hk.github.io/cardano-wallet/api/edge/#operation/signTransaction) endpoint.
```